init.js

```diff
@@ -62,7 +62,8 @@ function printUUID(metadataPath) {
     logger.info(`This deployment's identifier is ${uuidValue}`);
 }
 
-if (config.backends.data !== 'file' && config.backends.metadata !== 'file') {
+if (config.backends.data !== 'file' && config.backends.data !== 'multiple'
+    || config.backends.metadata === 'scality') {
```

Comment:
Wouldn't we still want to init if, for example, data = file/multiple and metadata = mem/scality?
Which would make it just:
(config.backends.data !== 'file' && config.backends.data !== 'multiple' && config.backends.metadata !== 'file')